tests/integration/customizations/s3/test_plugin.py

```diff
@@ -563,8 +563,8 @@ def test_copy_metadata_directive(self):
         # For comparing expires timestamp.
         self.override_parser(timestamp_parser=identity)
         bucket_name = _SHARED_BUCKET
-        original_key = 'foo.txt'
-        new_key = 'bar.txt'
+        original_key = random_chars(6)
+        new_key = random_chars(6)
```

Comment:
I know it's very unlikely that these will be duplicates, but why not just append a character to the new key to make sure?

Reply:
Updated.